VAMobile/src/components/LabelTag.tsx

```diff
@@ -96,9 +97,9 @@ const LabelTag: FC<LabelTagProps> = ({ text, labelType, onPress, a11yHint, a11yL
       name: 'Info',
       fill: 'tagInfoIcon',
       fill2: 'transparent',
-      height: 16,
-      width: 16,
-      mr: 10,
+      height: fontScale >= 2 ? 10 : 16,
+      width: fontScale >= 2 ? 10 : 16,
+      mr: fontScale >= 2 ? 5 : 10,
```

Comment:
Since `fontScale >= 2` is used in multiple places, that could be stored in a variable

Reply:
updated